sass/pages/_about.scss

```diff
@@ -0,0 +1,173 @@
+#about {
+  position: relative;
+  background: url(./../assets/images/code.jpg) no-repeat center;
+  background-color: #f4f4f4;
+}
+
+.about {
+  text-align: center;
+  padding: 20px;
+  padding-bottom: 40px;
+
+  &-principal {
+    position: relative;
+    z-index: 2;
```

Comment:
I think you inspire me to do more on SCSS. Good job:+1:

Reply:
It's awesome, isn't it?

<img src="https://media0.giphy.com/media/fT3PPZwB2lZMk/giphy.gif"/>